pennylane/_device.py

```diff
@@ -154,6 +156,12 @@ def wire_map(self):
         the wire labels used on this device"""
         return self._wire_map
 
+    @property
+    def execution(self):
+        """Number of times this device is executed by the evaluation of QNodes
+         running on this device"""
```

Comment:
A relatively minor suggestion, but it might be useful to provide the return type (`int`) within the docstring. However, for properties with a clear function it is usually not as essential.

Reply:
Makes sense! Docstring has been updated